common/shared.c

```diff
@@ -283,7 +283,8 @@ void ft_csusage(char *name, char *desc)
 
 	fprintf(stderr, "\nOptions:\n");
 	fprintf(stderr, "  -n <domain>\tdomain name\n");
-	fprintf(stderr, "  -p <port>\tnon default port number\n");
+	fprintf(stderr, "  -b <src_port>\tnon default source port number\n");
```

Comment:
Missed that.  `-b` isn't a deal breaker, just not my preference.  If @shefty is good with it, feel free to ignore me.